lib/main.dart

```diff
@@ -183,18 +218,31 @@ class _MyHomePageState extends State<MyHomePage> {
           child: Column(
             mainAxisAlignment: MainAxisAlignment.center,
             children: <Widget>[
-              Text(
-                AppLocalizations.of(context)!.tapToFindALanguagePartner,
-                style: Theme.of(context).textTheme.headlineSmall,
-                textAlign: TextAlign.center,
-              ),
-              const SizedBox(
-                height: 32,
+              Expanded(
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.end,
+                  children: [
+                    Text(
+                      callToActionText(timerDuration),
+                      style: Theme.of(context).textTheme.headlineSmall,
+                      textAlign: TextAlign.center,
+                      overflow: TextOverflow.visible,
```

Comment:
I don't know how yet to only scale text to a constrained container. I want to respect a user's A11y scale as much as possible while keeping the CTA button itself in a fixed position. Shazam doesn't respect A11y text scale at all. So even though the overflow goes behind the button, it's the best I can do right now.